scripts/testimonials.js

```diff
@@ -1,62 +1,36 @@
-const mySiema = new Siema({
-  selector: '.carousel--items',
-  duration: 200,
-  easing: 'ease-out',
-  perPage: 1,
-  startIndex: 0,
-  draggable: true,
-  multipleDrag: true,
-  threshold: 20,
-  loop: false,
-  rtl: false,
-  onInit: () => {},
-  onChange: () => {},
-})
+function setUpCarousel(root) {
+    const siema = new SiemaWithDots({
+        selector: root.querySelector('.carousel--items'),
```

Comment:
here we're using a specific DOM element, rather than a class, so that the function `setUpCarousel` is guaranteed to only set up a single carousel.